### PR TITLE
fix-small-things

### PR DESCRIPTION
**Pull request by tejaswini4**
- changed black, white to winner, looser in endGmae event
- validating player's turn is moved to othellofactory
- fixed the bug of being able to do passMove when it is not one's turn

### CODE REVIEW

File: contracts/contracts/othellofactory.sol [hunk 1]
```diff
@@ -174,17 +177,22 @@ contract othellofactory is Board{
             }
         }
         removeUsersFromGame();
-        emit EndGame(blackaddress,whiteaddress);
         
-        if(white==black){
-            return(blackaddress, true);
-        }
-        winner=black>white?blackaddress:whiteaddress;
+        _isDraw=white==black?true:false;
```
Comment (nickodell) [inline contracts/contracts/othellofactory.sol:181]: _isDraw=white==black?true:false;

This ternary expression is unnecessary. The result of `white==black` is already a boolean, so you could write:

    _isDraw = white == black;

Comment (tejaswini4) [reply contracts/contracts/othellofactory.sol:181]: yeah, you are right. Dumb me!

**Review by nickodell — APPROVED** [review]
LGTM.

File: contracts/contracts/othellofactory.sol [hunk 2]
```diff
@@ -174,17 +177,22 @@ contract othellofactory is Board{
...
     
+    // Returns the address of the player whose turn it is
+    function getCurrentTurnAddress() private view returns(address currentPlayer){
+        address currentTurnAddress=getMyGame().isWhiteTurn? getMyGame().whiteaddress: getMyGame().blackaddress;
+        return currentTurnAddress;
+    }
+    
     // Return the currentState of player's game
     function getCurrentState() public view returns (string memory state){
         if (activelyPlaying[msg.sender]==true){
```
Comment (virajshastri97) [inline contracts/contracts/othellofactory.sol:198]: This i think causes an error as the function is declared as view.. please remove view

**Pull request merged**